ownCloud/Tools/VendorServices.swift

```diff
@@ -54,6 +54,14 @@ class VendorServices : NSObject {
 		return false
 	}
 
+	var isEMMVersion: Bool {
+		if let isEMMVersion = Bundle.main.bundleIdentifier?.hasSuffix(".emm") {
```

Comment:
For consistency, this should just return the value of `OCLicenseEMMProvider.isEMMVersion`, so there's only one check to adjust if it ever needs to be changed.

Reply:
Thanks, fixed!